guide/mechanics/damageFormula.md

```diff
@@ -8,27 +8,35 @@ sidebar_label: Damage Formula
 
 ## The Formula
 
-Anthem's damage formula is as follows:
+It helps to break up the damage formula as follows
+
+```DamageDealt = NormalDamage * (1 + CriticalBonus) + 1```  
```

Comment:
I would leave out the +1 at the end. There's so much rounding crap happening along the way that answers are going to be slightly inaccurate anyway

Reply:
I'll try to address this as a comment. I think it's worth including since people are going to try and test it out.

Reply:
If you want to address it, maybe it's best to say in a comment "This may be off by 1 due to internal rounding in a number of places" and not complicate the formula